collapsible_section/collapsible.less

```diff
@@ -0,0 +1,53 @@
+@import '_vars.less';
+html {
+  font-size: 10px;
+}
+
+.collapsible-section {
+  font-family: @font-family-normal;
+
+  & > label {
+    padding: .7rem 1rem;
+    background-color: @blue;
+    color: white;
+    font-weight: bold;
+    font-family: 'Merriweather-Bold';
+    display: block;
+    cursor: pointer;
+    font-size: 1.6rem;
+
+    .fa-minus-square {
+      display: none;
+    }
+
+    .fa {
+      margin-right: .5rem;
+    }
+  }
+
+  & > section {
+    display: none;
+    font-size: 1.4rem;
+  }
+
+  .collapsible-section-handler:checked {
+   & ~ section {
+    display: block;
+  }
+
+  & ~ label .fa-minus-square {
+    display: inline;
+  }
+  & ~ label .fa-plus-square {
+    display: none;
+  }
+
+}
+
+  .collapsible-section-handler {
+    position: absolute;
```

Comment:
I feel like this is a thing we have done enough we should consider making it a helper.